main/tests/cypress/cypress/integration/project/grid/column/edit-column/add_columns_based_on_this_column.spec.js

```diff
@@ -12,7 +12,7 @@ describe(__filename, function () {
     cy.typeExpression('value.toLowercase()');
     cy.get(
       '.expression-preview-table-wrapper tr:nth-child(2) td:last-child'
-    ).contains('butter,with salt');
+    ).should('to.contain', 'butter,with salt');
```

Comment:
How is this fixing the text 😕 ?

Reply:
I am not sure, but it's working :-), when running 10x times locally I can have some flaky tests, even locally
With the new version I couldn't reproduce the flaky test